### PR TITLE
Update args to be compatible with polymer-cli

### PR DESCRIPTION
**Pull request by justinfagnani**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: src/args.ts [hunk 1]
```diff
@@ -12,20 +12,18 @@
...
   {
     name: 'version',
-    alias: 'v',
     description: 'Print version info',
     type: Boolean,
   },
   {
     name: 'root',
-    description: 'The root directory. Defaults to cwd',
+    description: 'The directory where your project exists. Defaults to the ' +
```
Comment (rictic) [inline src/args.ts]: `The directory where your project exists` is kinda roundabout and not super clear. What about:

```javascript
'The root directory of your project. Defaults to the current working directory'
```

Comment (justinfagnani) [inline src/args.ts]: This has to match what's in the CLI, or the validation fails.

Comment (justinfagnani) [inline src/args.ts]: I'd like to come up with a better way to share argument definitions across repos, but not today :)

Comment (justinfagnani) [inline src/args.ts]: Actually, since I'm about to push a PR on CLI, I can change this now.

**Pull request merged**